inject_test.go

```diff
@@ -1,8 +1,7 @@
-package inject_test
+package inject
```

Comment:
Same comment as above - I'm not at all questioning that this is a good change, but ideally we'd merge this fork upstream and make the change there, or make the change upstream and them pull it into the fork.

Maybe I should be asking: Is this intended to be a permanent fork? If so, what change are we making that requires the fork? Ideally, that should be the _only_ change made in the forked repo, so that it's as easy as possible to pull in upstream improvements.